tests/Supportive/DependencyInjection/DeptracExtensionTest.php

```diff
@@ -32,6 +32,14 @@ final class DeptracExtensionTest extends TestCase
         ],
     ];
 
+    private array $analyserDefaults = [
```

Comment:
Should this be a `const` instead?

Reply:
I was following the example of the field above.

Reply:
Should I make both of them const?

Reply:
I am not invested either way, just looked unintuitive to me when I saw it in the PR, that's why I mentioned it.